react-live.css

```diff
@@ -92,3 +92,9 @@
 .token.deleted {
   color: red;
 }
+
+.tabGuarded{
```

Comment:
The outline code is fine. The user should be able to deal with the details of how it should look :)

But can we put it at the top of the file, rename it to `.tab-guarded`, and make the rule more specific please (i.e. `.prism-code.tab-guarded`)? Cheers :)